stdeb/util.py

```diff
@@ -1340,13 +1344,13 @@ def build_dsc(debinfo,
 
     #    J. debian/source/format
     os.mkdir(os.path.join(debian_dir,'source'))
-    fd = open( os.path.join(debian_dir,'source','format'), mode='w')
-    fd.write('3.0 (quilt)\n')
-    fd.close()
+    with open(os.path.join(debian_dir,'source','format'), mode='w') as fd:
+        fd.write('3.0 (quilt)\n')
```

Comment:
Please revert unrelated changes like this. Consider creating a separate PR instead.